src/stylesheets/components/_search.scss

```diff
@@ -11,69 +7,59 @@ $usa-btn-big-width:     11.6rem;
 
 .usa-search {
   @include clearfix;
-  max-width: none;
+  @include typeset;
```

Comment:
Since you have the new settings var, use `typeset($theme-font-family-search)` here instead

Reply:
Done in 355a34e.